test/__snapshots__/index.test.js.snap

```diff
@@ -1631,14 +1635,14 @@ modern
 
 
 Build \\"modernLib\\" to dist:
-159 B: modern-lib.modern.js.gz
```

Comment:
The difference in size is noticeable

Reply:
(this one mostly comes down to rest parameters being supported in preset-modules but transpiled to arguments access in preset-env)